host/src/PlaceholderApp/JitTrace/coldstart.jittrace

```diff
@@ -0,0 +1,331 @@
+[S.P.CoreLib]System.Array.Empty<Candidate>()~System.Array,System.Private.CoreLib~0~1:Microsoft.AspNetCore.Routing.Matching.Candidate,Microsoft.AspNetCore.Routing~Empty
```

Comment:
@eliaslopezgt please replace `coldstart.jittrace` and `linux.coldstart.jittrace` files.